tardis/model/tests/test_base.py

```diff
@@ -78,9 +79,9 @@ def setup(self, example_model_file_dir):
         self.simulation_state = SimulationState.from_config(self.config)
 
     def test_velocities(self):
-        assert self.simulation_state.v_inner.unit == u.Unit("cm/s")
```

Comment:
I notice that the default unit appeared to be cm/s. See previous comment regarding km. Not sure if we want to "pick" one or the other as long as we convert properly.

Reply:
remove defaults?

Reply:
cgs defaults make sense to me

Reply:
but we are converting - so there is nothing to be done here - correct?